test/inductor/test_aot_inductor.py

```diff
@@ -670,10 +683,11 @@ def forward(self, x, y):
         ):
             self.check_model(Repro(), example_inputs)
 
-    @unittest.skipIf(
-        torch.cuda.device_count() < 2, "The test requires multiple devices"
-    )
+    @requires_multigpu()
     def test_non_default_cuda_device(self):
+        if self.device == "cpu":
+            raise unittest.SkipTest("requires CUDA")
```

Comment:
Wondering if ```@requires_cuda()``` decorator would work or not?

Reply:
That will be a problem when running on a gpu instance, where both cuda and cpu version of this test will be run.